readers/elasticsearch/src/main/resources/template.json

```diff
@@ -1157,6 +1157,12 @@
     "bytes_consumed_rate": {
       "type": "double",
       "index": false
+    },
+    "rdd_name": {
+      "type": "keyword",
+      "norms": false,
+      "index_options": "freqs",
+      "eager_global_ordinals": true
     }
```

Comment:
could you please add schema for all new fields

Reply:
@Willymontaz : You didn't address this comment, did you?

Reply:
No I did not